metrics/versions.py

```diff
@@ -172,15 +167,17 @@ def assess_next_release_risk(session, configuration: Configuration, project_id:i
     # # Filter our dataframe based on condition
     # filtered_df = df[condition]
 
-    bug_velocity = np.array(df['bug_velocity'])
+    bugs = np.array(df['bugs'].to_numpy())
```

Comment:
I meant only `df["bugs"].to_numpy()` since it returns a ndarray already, no need to call np.array on top